flang/docs/Directives.md

```diff
@@ -29,3 +29,28 @@ A list of non-standard directives supported by Flang
     end
   end interface
 ```
+
+## ARM Streaming SVE directives
+
+These directives are added to support ARM specific instructions. All of
+these attributes apply to a specific subroutine or function. These directives
+are identical to the attributes provided in C and C++ for the same purpose.
+See https://arm-software.github.io/acle/main/acle.html#controlling-the-use-of-streaming-mode for more in depth details. (For the following, function is used
+to mean both subroutine and function).
+
+### Directives relating to ARM Streaming mode
+
+* `!dir$ arm_streaming` - The function is intended to be used in streaming
+  mode.
+* `!dir$ arm_streaming_compatible` - The function can work both in streaming
+  mode and non-streaming mode.
+* `!dir$ arm_streaming` - The function will enter streaming mode, and return to
```

Comment:
Did you mean `arm_locally_streaming`?

Reply:
Yes, of course. Typical copy-pasta error. Thanks for pointing it out.